src/components/totals/TotalHours/index.js

```diff
@@ -43,17 +43,34 @@ module.exports = React.createClass({
 
   componentWillMount: function() {
     this.setState({ isLoading: true });
-    campaigns.find(this.props.campaignUid, this.onSuccess);
+    campaigns.find(this.props.campaignUid, this.combineActivityData);
   },
 
-  onSuccess: function(result) {
-    var fitnessResults = result.campaign.fitness_activity_overview.run;
+  combineActivityData: function(result) {
+    var fitnessActivity = result.campaign.fitness_activity_overview;
 
-    if (fitnessResults){
+    if (fitnessActivity !== null) {
```

Comment:
Hmm, I'll go over this with you tomorrow. We can use _.reduce to simplify this a lot. :-)